faiss/python/setup.py

```diff
@@ -16,17 +16,29 @@
 shutil.copyfile("__init__.py", "faiss/__init__.py")
 shutil.copyfile("loader.py", "faiss/loader.py")
 shutil.copyfile("swigfaiss.py", "faiss/swigfaiss.py")
```

Comment:
Should this be guarded by an `if os.path.exists` as well?

Reply:
`swigfaiss.py` is always generated regardless of FAISS_OPT_LEVEL.